plugins/chat/app/jobs/regular/chat/notify_watching.rb

```diff
@@ -15,6 +15,17 @@ def execute(args = {})
         @is_direct_message_channel = @chat_channel.direct_message_channel?
 
         always_notification_level = ::Chat::UserChatChannelMembership::NOTIFICATION_LEVELS[:always]
+        thread_watcher_ids = []
```

Comment:
Do we need to keep track of `thread_watcher_ids` in Ruby-land? 🤔 

Can we instead use a CTE that would avoid passing rows between the database and Ruby?